src/code/game.c

```diff
@@ -281,6 +282,14 @@ void GameState_Update(GameState* gameState) {
 
     func_800C4344(gameState);
 
+#if OOT_VERSION < PAL_1_0
+    if (SREG(48) != 0) {
```

Comment:
looks like sreg 48 is `R_VI_MODE_EDIT_STATE`, but unsure if it has the same meaning here. at first glance it seems like it does

Reply:
good catch thanks